cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetector.java

```diff
@@ -48,7 +50,7 @@
   static final String METRIC_REGISTRY_NAME = "AnomalyDetector";
   private static final int INIT_JITTER_BOUND = 10000;
   private static final long SCHEDULER_SHUTDOWN_TIMEOUT_MS = 5000L;
-  private static final int NUM_ANOMALY_DETECTION_THREADS = 5;
+  private static final int NUM_ANOMALY_DETECTION_THREADS = 6;
```

Comment:
Nit: Can we make it `KafkaAnomalyType.cachedValues().size() + 1` and add a comment to clarify why we have a `+ 1`?